src/stylesheets/book.css

```diff
@@ -0,0 +1,179 @@
+.books-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.book-container {
+  margin-top: 20px;
+  display: flex;
+  justify-content: space-between;
+  padding: 30px 10px;
+  width: 100%;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+.wrapper {
+  width: 85%;
+}
+
+.book-wrapper {
+  display: flex;
+  justify-content: space-between;
+}
+
+.book-left {
+  width: 45%;
+}
+
+.book-right {
+  padding-right: 2.5em;
+}
+
+.book-right > * {
+  margin: 5px 0;
+}
+
+.book-category {
+  opacity: 0.5;
+  font-size: 14px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
```

Comment:
- [ ] These CSS rules are being repeated a lot, please create a class that contains them so you can reuse them easily wherever you need them instead of repeating them in every class needed.